### PR TITLE
Add a page about the metrics view

### LINKED ISSUE #220: Add a page about the p95, p99 and median in the metrics tab
Related to Scalingo/autoscaler-service#1

### PR DESCRIPTION
**Pull request by EtienneM**
Fix #220

### CODE REVIEW

File: _posts/app/2018-02-07-metrics.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: Applications Metrics
+modified_at: 2018-02-07 00:00:00
+category: app
+tags: app metrics
+---
+
+We gather a few performance metrics of your application that are accessible on the dashboard of your
```
Comment (Soulou) [inline _posts/app/2018-02-07-metrics.md]: Not `We`, `Performance metrics from applications are gathered and are accessible from the dashboard`

File: _posts/app/2018-02-07-metrics.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+application. Just head to the Metrics tab of your application to see charts of these key metrics.
+
+## Global View
+
+This page contains two main parts: **application charts** and **containers charts**. You can adjust
+the viewing period to 3h, 12h, 24h, 48h and 72h in the past. We try to autodetect your timezone from
```
Comment (Soulou) [inline _posts/app/2018-02-07-metrics.md]: Passive

File: _posts/app/2018-02-07-metrics.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+The application chart displays global data that are not container specific: events and routing
+metrics.
+
+The **events chart** is here to show you all the events that happened during the viewing period.
+This can help you link an application behaviour with events that happened on the platform, e.g. spot
+a deployment that contains a memory leak or follow your application behaviour after a scale up (or
+down).
+
+A lot of events are available on your application timeline but we think that only a few are relevant
```
Comment (Soulou) [inline _posts/app/2018-02-07-metrics.md]: Passive

File: _posts/app/2018-02-07-metrics.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+### Container Charts
+
+{% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_dashboard_metrics_containers.png" %}
+{% include mdl_img.html %}
+
+The container charts use the container types defined in your [Procfile]({% post_url internals/2014-12-01-procfile %}).
+
+For each of your container type, two charts are shown. The first one shows the **CPU usage** and the
+second one the **Memory** and **swap** usage of this type of container. For the memory chart, we
```
Comment (Soulou) [inline _posts/app/2018-02-07-metrics.md]: Passive

File: _posts/app/2018-02-07-metrics.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+on the metrics view:
+
+- Restart event
+- Deploy event
+- Scale event
+
+You will find the complete list of events in our [developer
+documentation](https://developers.scalingo.com/events.html)).
+
+The **requests per minute** is here to show you how many requests your application gets per minute. It
```
Comment (Soulou) [inline _posts/app/2018-02-07-metrics.md]: `is here to show how many requests are received by the application`

Also try to remove a maximum of `you/your`, it's not really beautiful/formal

**Comment by Soulou** [comment]
screenshot from 2018-02-07 13-35-50

File: _posts/app/2018-02-07-metrics.md [hunk 6]
```diff
@@ -22,30 +22,29 @@ your browser and you can still change it later if needed.
...
-This can help you link an application behaviour with events that happened on the platform, e.g. spot
-a deployment that contains a memory leak or follow your application behaviour after a scale up (or
-down).
+The **events chart** is here to show all the events that happened during the viewing period. This
+can help you link an application behaviour with events that happened on the platform, e.g. spot a
+deployment that contains a memory leak or follow your application behaviour after a scale operation.
 
-A lot of events are available on your application timeline but we think that only a few are relevant
-on the metrics view:
+A lot of events are available on an application timeline but only a few relevant are displayed on
```
Comment (Soulou) [inline _posts/app/2018-02-07-metrics.md]: `on the application timeline` ?

File: _posts/app/2018-02-07-metrics.md [hunk 7]
```diff
@@ -22,30 +22,29 @@ your browser and you can still change it later if needed.
...
+the metrics view:
 
 - Restart event
 - Deploy event
 - Scale event
 
-You will find the complete list of events in our [developer
+A complete list of events is available on our [developer
```
Comment (Soulou) [inline _posts/app/2018-02-07-metrics.md]: Is it really `on our documentation` ? It sounds weird

File: _posts/app/2018-02-07-metrics.md [hunk 8]
```diff
@@ -5,14 +5,14 @@ category: app
...
+Performance metrics of applications hosted on Scalingo are gathered and accessible on the web
+dashboard. Just head to the Metrics tab of your application to see charts of these key metrics.
 
 ## Global View
 
-This page contains two main parts: **application charts** and **containers charts**. You can adjust
-the viewing period to 3h, 12h, 24h, 48h and 72h in the past. We try to autodetect your timezone from
-your browser and you can still change it later if needed.
+This page contains two main parts: **application charts** and **containers charts**. The viewing
+period is adjustable to 3h, 12h, 24h, 48h and 72h in the past. We try to autodetect your timezone
```
Comment (Soulou) [inline _posts/app/2018-02-07-metrics.md]: `An attempt to autodetect the local timezone is done but it is configurable` ?

File: _posts/app/2018-02-07-metrics.md [hunk 9]
```diff
@@ -56,24 +55,24 @@ in millisecond.
 
 The container charts use the container types defined in your [Procfile]({% post_url internals/2014-12-01-procfile %}).
 
-For each of your container type, two charts are shown. The first one shows the **CPU usage** and the
-second one the **Memory** and **swap** usage of this type of container. For the memory chart, we
-stack your memory usage (in blue) and your SWAP usage (in red). That way you can easily see the
-total memory usage of your application.
+For each container type, two charts are shown. The first one shows the **CPU usage** and the second
+one the **Memory** and **swap** usage of this type of container. For the memory chart, the memory
```
Comment (Soulou) [inline _posts/app/2018-02-07-metrics.md]: `Memory` -> `memory`?

**Comment by EtienneM** [comment]
@Soulou OK to merge?

**Comment by yannski** [comment]
@EtienneM can you review where this file should be placed regarding the doc organization?

**Comment by EtienneM** [comment]
@yannski reviewable

**Comment by yannski** [comment]
@EtienneM can you rebase? I'm pretty sure a link will be broken :)

**Pull request merged**